### DISCUSSION TITLE
Please Clarify few doubts

### DISCUSSION THREAD

**Discussion by Mahesh-Gunda-Maersk** [stackrox/kube-linter#466] [Q&A]
Found this great tool and am interested to explore more and use in CI/CD Process. 
1.  by default which version of k8s the YAML validation happens? (is it 1.22.0 or 1.23 or 1.19?) 
2.  does it support validation against specific versions like --schema-version 1.24.8?  
3.  if not when can we expect this feature to be available? 
4.  when I tested the argocd application/project YAML files, Kube-linter is unable to recognize them.  Hope this is also in pipeline. 

Please answer.  Thanks for your time.

**Comment by janisz** [comment]
Thank you for considering kube-linter. By default, kube-linter attempts to support all versions of k8s simultaneously without specifying a single version for validation. If you have encountered different behavior between versions, please provide more information and examples so we can improve it.

At this time, kube-linter does not support validation against specific versions using the --schema-version flag. We currently do not have plans to implement this feature in the near future, but we would appreciate your feedback on why this feature would be important for your use case.

> 4. when I tested the argocd application/project YAML files, Kube-linter is unable to recognize them. Hope this is also in pipeline.
Could you make a link to yaml specifications. Is it k8s or argocd config? kube-linter supports only k8s yamls.

Thank you again for your interest in kube-linter, and please let us know if you have any further questions or feedback.
